### PR TITLE
bilbilly binary encoding bug fix + alt bilbilly training setup + training script cleanup

### PR DESCRIPTION
**Pull request by jaycehaoranwang**
fixed binary sex encoding forward pass bug for bilbilly
main training script cleanup 
using alternate training setup by default with AdamW instead of Adam optimizer and changed lr scheduler hyperparams

### CODE REVIEW

File: training/main_training_script.py [hunk 1]
```diff
@@ -213,8 +215,8 @@ def main():
     # default_out_features = trainingModel.fc.out_features
     # trainingModel.fc = nn.Sequential(trainingModel.fc, nn.ReLU(), nn.Linear(in_features=default_out_features, out_features=1, bias=True))
     trainingModel = trainingModel.to(device)
-    optimizer = torch.optim.Adam(trainingModel.parameters(),lr=hyperparams['optimizer_lr']) 
-    scheduler = lr_scheduler.ReduceLROnPlateau(optimizer,mode='min',factor=0.5, patience=10,min_lr=0.001, verbose=True) #Start at large learning rate to quickly learn and only reduce when loss plateaus
+    optimizer = torch.optim.AdamW(trainingModel.parameters(),lr=hyperparams['optimizer_lr']) 
+    scheduler = lr_scheduler.ReduceLROnPlateau(optimizer,mode='min',factor=0.1, patience=10,min_lr=0.0002, verbose=True) #Start at large learning rate to quickly learn and only reduce when loss plateaus
```
Comment (lolzballs) [inline training/main_training_script.py:219]: shouldn't we put these in the hyperparams.yml?

**Review by lolzballs — APPROVED** [review]

File: training/main_training_script.py (hunk 1 above)
Comment (jaycehaoranwang) [reply training/main_training_script.py:219]: o you are right I will do that in another PR at some point
